### PR TITLE
add Clover Theme Manager

### PR DESCRIPTION
**Pull request by ye4241**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/clover-theme-manager.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "clover-theme-manager" do
```
Comment (vitorgalvao) [inline Casks/clover-theme-manager.rb:1]: No hyphens.

File: Casks/clover-theme-manager.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  version "0.77.2"
+  sha256 "6cd6247d407f36a852491b6cc98b279d194952cee066bd1406ce9b593cf7af57"
+
+  url "https://sourceforge.net/p/cloverefiboot/themes/ci/master/tree/CloverThemeManagerApp/Updates/CloverThemeManager.zip?format=raw"
```
Comment (vitorgalvao) [inline Casks/clover-theme-manager.rb:5]: See other sourceforge casks and follow their pattern.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by yurikoles** [comment]
From official [thread](https://www.insanelymac.com/forum/topic/302674-clover-theme-manager/):
>Clover Theme Manager has now been retired
>This is due to the lack of time to maintain and develop any further
>Please see Clover.app instead which includes a theme manager

The problem of obtaining URL in official format is that it was never released in files section, it's a binary that's committed to Git repo.

**Comment by vitorgalvao** [comment]
If it’s been discontinued, we’d rather not add it.

**Pull request closed without merging**